servo

```diff
@@ -443,6 +443,9 @@ def exec_command():
         p = cmd['param']
         try:
             v = measure(p)
+        except DriverError as e:
+            traceback.print_exc()
```

Comment:
This traceback seems redundant. DriverError is raised deliberately and doesn't benefit from a stack trace in the logs. My guess is that it was added here as a copy-paste from the 'ADJUST' case below. It isn't needed there, either - both can be safely removed.

Reply:
Good call, looks like the adjust case was copy pasted as well from the generic Exception handler where a traceback makes much more sense